proc/proc-lib.pl

```diff
@@ -249,6 +249,53 @@ sub safe_exec_reaper
 	} while($xp > 0);
 }
 
+
+# linux_openpty()
+# Linux-only, pure-Perl openpty(3)-style helper.
+# Returns master fh, slave fh, slave path, and ioctl value on success
+sub linux_openpty
```

Comment:
if this is Linux specific, could you move it to proc/linux-lib.pl ?

This also removes the need for line 258.

Reply:
Agreed! Fixed now!